scripts/libme.sh

```diff
@@ -0,0 +1,7 @@
+download_and_extract https://github.com/NT-Bourgeois-Iridescence-Technologies/MElib/archive/master.zip MElib
+make
```

Comment:
This step currently fails for me with the following error:
```
make -f MakefileLib all
make[1]: Entering directory '/build/new/psplibraries/build/MElib-master'
psp-gcc -I/include -I./include -I./libs/include -I. -I/usr/local/pspdev/psp/sdk/include -G0 -O3 -D_PSP_FW_VERSION=150   -c -o src/me.o src/me.c
psp-gcc -I/include -I./include -I./libs/include -I. -I/usr/local/pspdev/psp/sdk/include -G0 -O3 -I/include -I./include -I./libs/include -I. -I/usr/local/pspdev/psp/sdk/include -G0 -O3   -c -o src/MediaEngine.o src/MediaEngine.S
psp-gcc -I/include -I./include -I./libs/include -I. -I/usr/local/pspdev/psp/sdk/include -G0 -O3 -D_PSP_FW_VERSION=150   -c -o src/melib.o src/melib.c
src/melib.c: In function 'J_Init':
src/melib.c:72:6: warning: assignment discards 'volatile' qualifier from pointer target type [-Wdiscarded-qualifiers]
   72 |  mei = (volatile struct me_struct*)malloc_64(sizeof(struct me_struct));
      |      ^
src/melib.c:73:6: warning: assignment discards 'volatile' qualifier from pointer target type [-Wdiscarded-qualifiers]
   73 |  mei = (volatile struct me_struct*)( ((u32)(mei)) | 0x40000000);
      |      ^
psp-gcc -I/include -I./include -I./libs/include -I. -I/usr/local/pspdev/psp/sdk/include -G0 -O3 -D_PSP_FW_VERSION=150  -L./libs/lib/ -L. -L/usr/local/pspdev/psp/sdk/lib   src/me.o src/MediaEngine.o src/melib.o -lpsprtc -lpspsdk -lpspdebug -lpspdisplay -lpspge -lpspctrl -lpspsdk -lc -lpspnet -lpspnet_inet -lpspnet_apctl -lpspnet_resolver -lpsputility -lpspuser -lpspkernel -o LibME.elf
/usr/local/pspdev/lib/gcc/psp/9.3.0/../../../../psp/lib/crt0.o: In function `_main':
(.text+0x104): undefined reference to `main'
collect2: error: ld returned 1 exit status
/usr/local/pspdev/psp/sdk/lib/build.mak:176: recipe for target 'LibME.elf' failed
make[1]: *** [LibME.elf] Error 1
make[1]: Leaving directory '/build/new/psplibraries/build/MElib-master'
makefile:11: recipe for target 'build_game' failed
make: *** [build_game] Error 2
Failed installing libme!
```

Reply:
It's now make -f MakefileLib to fix it.